production_loader.py

```
@@ -0,0 +1,1053 @@
+            )
+        )
+
+    def _update_person_references(self, session, old_id: int, new_id: int) -> None:
+        reference_tables = [
+            ("unified_transaction_persons", "person_id"),
+            ("unified_committee_persons", "person_id"),
+            ("unified_campaigns", "candidate_person_id"),
+        ]
+        for table, column in reference_tables:
+            session.exec(
+                text(f"UPDATE {table} SET {column} = :new WHERE {column} = :old").bindparams(
```

Ruff (0.14.7)

Possible SQL injection vector through string-based query construction

(S608)